src/main/java/robot/subsystems/hatchIntake/commands/FoldCommand.java

```diff
@@ -0,0 +1,41 @@
+package robot.subsystems.hatchIntake.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import robot.Robot;
+
+public class FoldCommand extends Command {
```

Comment:
I would make this command an [InstantCommand](http://first.wpi.edu/FRC/roborio/beta/docs/java/edu/wpi/first/wpilibj/command/InstantCommand.html).